content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -467,6 +467,41 @@ Other alternatives for enforcing policies are being developed in the Kubernetes
 - [Kyverno](https://kyverno.io/policies/pod-security/)
 - [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
 
+## Pod OS field
+
+Windows in Kubernetes has some limitations and differentiators from standard Linux-based
+workloads. Specifically, many of the Pod SecurityContext fields
+[have no effect on Windows](/docs/concepts/windows/intro/#compatibility-v1-pod-spec-containers-securitycontext).
+
+### Pod API validation changes
```

Comment:
```suggestion
### Pod security checks based on Pod operating system
```

Document the behavior, _not_ the change in behavior (that's for the release notes)/